index.js

```diff
@@ -1,31 +1,117 @@
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    featured_image: 'images/Snapshoot Portfolio_1.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    link_to_live: 'https://ibrahim777764.github.io/new_portfolio/',
+    link_to_source: 'https://github.com/ibrahim777764/new_portfolio/',
+  },
+  ];
+
+  const gridTemplate = document.getElementById('grid-template');
+  const recentWorks = document.getElementById('Recent-id');
+  const detailsPopUpHeading = document.querySelector('.detail-popup-heading');
+  const popupWindow = document.getElementById('mobile-details-modal');
+  projects.forEach((project) => {
+    const clonedNode = gridTemplate.children[0].cloneNode(true);
+    clonedNode.children[0].children[0].innerText = project.name;
+    recentWorks.appendChild(clonedNode);
+
+    detailsPopUpHeading.textContent = project.name;
+
+    const btnPopups = recentWorks.querySelectorAll('.btn-popup');
+    btnPopups.forEach((button) => {
+      button.addEventListener('click', () => {
+        popupWindow.classList.remove('hidden');
+      });
+    });
+  });
```

Comment:
- [ ] As the previous reviewer mentioned, make sure that the text and icon are beside each other. They are still wrapping: 
<img width="277" alt="Screenshot 2022-03-10 at 13 54 32" src="https://user-images.githubusercontent.com/77038610/157637022-f74b1345-9805-4cd7-943c-072ad3d15743.png">

Try this:
```
button {
font-size: 15px;
letter-spacing: 0px;
}
```